lib/private/Security/RateLimiting/Backend/IBackend.php

```diff
@@ -35,16 +35,14 @@
  */
 interface IBackend {
 	/**
-	 * Gets the amount of attempts within the last specified seconds
+	 * Gets the amount of attempts for the specified method
```

Comment:
```suggestion
	 * Gets the number of attempts for the specified method
```

if we're making things more clear :)